posthog/hogql/database/schema/events.py

```diff
@@ -85,6 +86,16 @@ class EventsTable(Table):
         # These are swapped out if the user has PoE enabled
         "person": FieldTraverser(chain=["pdi", "person"]),
         "person_id": FieldTraverser(chain=["pdi", "person_id"]),
+        "$group_0": StringDatabaseField(name="$group_0"),
```

Comment:
This field is actually the same as `properties.$group_0`. The `$group_0` columns are materialized columns, calculated directly from the property. If we specify `properties.$group_0`, it'll resolve to the column itself. Other materialized properties start with `mat_`, but the group ones are "top level" for some reason. I'm not sure if it makes sense to try linking to the property instead of adding a column or not. Fine to keep as is.

Reply:
> but the group ones are "top level" for some reason.

That's because these are explicitly generated and expected to exist in every installation: https://github.com/PostHog/posthog/blob/master/posthog/models/event/sql.py#L59

whereas the ones starting with `mat_` are autogenerated/materialised/not necessarily always there, which is what prompts the sanity check that you shouldn't be explicitly querying for a `mat_XYZ` property unless you're very sure it will exist.

or so I think was the intent at the time 😅